mount_shares.py

```diff
@@ -207,12 +205,12 @@ def mount(shares):
                 #smbClient.createMountPoint( smbClient, directory, hostname)
                 """
                 if not options.write:
-                    mountCommand = 'mount -r -t cifs //'+ipDirectory+' ./"'+hostnameDirectory+'" -o username='+username+',password=\''+password+'\''
+                    mountCommand = 'mount -r -t cifs //'+ipDirectory+' ./"'+hostnameDirectory+'" -o credentials='+full_path
```

Comment:
So this works for auth files but breaks the mount command when users have a password on the command line as its not being passed in an auth file. So mount won't know where to go to get the actual creds.